common/test/acceptance/pages/lms/bookmarks.py

```diff
@@ -53,11 +54,32 @@ def breadcrumbs(self):
         breadcrumbs = self.q(css=self.BOOKMARKED_BREADCRUMBS).text
         return [breadcrumb.replace('\n', '').split('-') for breadcrumb in breadcrumbs]
 
-    def click_bookmark(self, index):
+    def click_bookmarked_unit(self, index):
         """
-        Click on bookmark at index `index`
+        Click on bookmarked unit at index `index`
 
         Arguments:
             index (int): bookmark index in the list
         """
         self.q(css=self.BOOKMARKED_ITEMS_SELECTOR).nth(index).click()
+
+    def bookmark_button_visible(self):
+        """ Check if bookmark button is visible """
+        EmptyPromise(lambda: self.q(css='.bookmark-button').visible, "Bookmark button visible").fulfill()
+        return True
+
+    @property
+    def bookmark_button_bookmarked(self):
+        """ Return `bookmarked` if button is in bookmarked state else `un-bookmarked` """
+        return 'bookmarked' if self.q(css='.bookmark-button.bookmarked').present else 'un-bookmarked'
+
+    @property
+    def bookmark_icon_visible(self):
+        """ Check if bookmark icon is visible on sequence item """
+        return self.q(css='.bookmark-icon').visible
```

Comment:
Is this the bookmark icon for the current unit? Or any bookmark icon on the sequential?